bin/sam-translate.py

```diff
@@ -5,15 +5,25 @@
 Known limitations: cannot transform CodeUri pointing at local directory.
 
 Usage:
-  sam-translate.py --input-file=sam-template.yaml [--output-file=<o>]
+  sam-translate.py --template-file=sam-template.yaml [--verbose] [--output-template=<o>]
+  sam-translate.py package --template-file=sam-template.yaml --s3-bucket=my-bucket [--verbose] [--output-template=<o>]
```

Comment:
Can you confirm you've manually tested these new commands with this latest version of the fix before merging?